index.html

```diff
@@ -786,9 +786,9 @@ <h3 class="title-wrapper center card-header"><a href="https://github.com/ranelpa
                         return;
                     }
 
-                    // Remove spaces at both ends of the phrase
+                    // Remove spaces at starting of the phrase
                     // which might be accidentally typed.
-                    var typedPhrase = this.typedPhrase.trim();
+                    var typedPhrase = this.typedPhrase.replace(/^ +/, "");
```

Comment:
Please use `.trimStart()` which is simpler/more readable:
`var typedPhrase = this.typedPhrase.trimStart()`